src/css/typography.css

```diff
@@ -8,6 +8,11 @@
   src: url("../fonts/Gilroy-Bold.ttf");
 }
 
+@font-face {
+  font-family: "gilroy-semi-bold";
```

Comment:
Which PR are you planning solving the font weight issue in?

Reply:
In tbis PR: https://github.com/xmartlabs/xl-blog/pull/66